code/modules/clothing/head/misc.dm

```diff
@@ -300,7 +300,7 @@
 	item_state = "helmet"
 
 /obj/item/clothing/head/fedora
-	name = "\improper fedora"

```

Comment:
Тут оно как раз таки лишнее, ведь:
>By default, if the object name is capitalized, it is assumed to be proper, and otherwise it is assumed to be improper.

Фактически можно убирать импроперы у всего, что начинается не с заглавной.

Если конечно бульёнд не врёт.